### PR TITLE
Fix the new ios version format "iOS x.x.x"

### PR DESCRIPTION
**Pull request by TetrixGauss**
The new version string preprocessing logic that removes 'iOS ' prefix should have test coverage to verify it correctly handles edge cases like versions that don't start with 'iOS ', versions with different casing, or malformed version strings.

### CODE REVIEW

File: lib/src/upgrade_store_controller.dart [hunk 1]
```diff
@@ -13,19 +13,13 @@ import 'upgrader_version_info.dart';
 
 abstract class UpgraderStore {
   Future<UpgraderVersionInfo> getVersionInfo(
-      {required UpgraderState state,
-      required Version installedVersion,
-      required String? country,
-      required String? language});
+      {required UpgraderState state, required Version installedVersion, required String? country, required String? language});
```
Comment (larryaasen) [inline lib/src/upgrade_store_controller.dart]: @TetrixGauss Why is this line so long? I think the line limit is 80 characters.

**Review by larryaasen — CHANGES_REQUESTED** [review]
@TetrixGauss Why did you change these lines when nothing actually changed? We don't accept code changes containing only formatting changes.

**Comment by larryaasen** [comment]
@TetrixGauss Any update on this?
